extensions/positron-python/python_files/posit/positron/positron_ipkernel.py

```diff
@@ -220,11 +221,48 @@ def connection_show(self, line: str) -> None:
 original_showwarning = warnings.showwarning
 
 
+class PositronDisplayFormatter(DisplayFormatter):
+    @traitlets.default("ipython_display_formatter")
+    def _default_formatter(self):
+        return PositronIPythonDisplayFormatter(parent=self)
+
+
+class PositronIPythonDisplayFormatter(IPythonDisplayFormatter):
+    print_method = traitlets.ObjectName("_ipython_display_")
+    _return_type = (type(None), bool)
+
+    @catch_format_error
+    def __call__(self, obj):
+        """Compute the format for an object."""
+        try:
```

Comment:
Should this also be wrapped in `if self.enabled`? I'm not really sure how these work.

Reply:
Looking into this! I am also seeing different plots than anticipated for the retina setting in plotnine, which is maybe related 👀